src/DataTable/index.jsx

```diff
@@ -20,15 +20,17 @@ import TableHeaderCell from './TableHeaderCell';
 import TableCell from './TableCell';
 import TableHeaderRow from './TableHeaderRow';
 import TablePagination from './TablePagination';
+import DataTableContext from './TableContext';
 
 function DataTable({
-  columns, data, bulkActions, defaultColumnValues, additionalColumns, isSelectable,
+  columns, data, defaultColumnValues, additionalColumns, isSelectable,
   isPaginated, manualPagination, pageCount, itemCount,
   isFilterable, manualFilters, fetchData, initialState,
   isSortable, manualSortBy,
   initialTableOptions,
   EmptyTableComponent,
-  numBreakoutFilters,
+  children,
+  ...rest
```

Comment:
If DataTable consumed a DataTableContext, then you could reduce these props to ones mostly about display and composition of children.